### PR TITLE
Secure Cookie flag in JS SDK

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated the JS SDK docs to include mention of the secureCookie flag in the `load` section.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx [hunk 1]
```diff
@@ -85,6 +85,25 @@ rudderanalytics.load(<WRITE_KEY>, <DATA_PLANE_URL>, {
...
+
+RudderStack lets you set a `secureCookie` flag while loading the SDK. This flag allows you to control how you send the cookie to the storage backend, i.e. via HTTP or HTTPS.
+
+An example is shown below:
+
+```javascript
+rudderanalytics.load(WRITE_KEY, DATA_PLANE_URL, {
+  configUrl: string, // defaults to https://api.rudderlabs.com
+  logLevel: "DEBUG",
+  secureCookie: true
```
Comment (saikumarrs) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx:98]:
```suggestion
  secureCookie: true // defaults to false
```

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx [hunk 2]
```diff
@@ -85,6 +85,25 @@ rudderanalytics.load(<WRITE_KEY>, <DATA_PLANE_URL>, {
   More information on how to get the <code class="inline-code">CONTROL_PLANE_URL</code> can be found <a href="https://docs.rudderstack.com/get-started/control-plane-lite/#what-is-the-control-plane-url">here</a>.
 </div>
 
+### Secure cookie flag
```
Comment (saikumarrs) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx:88]: Can we merge this to the "The options parameter" section as we are trying to keep the documentation as short as possible?

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx:88]: I placed it here for immediate visibility. Maybe we can add another reference in that section? @saikumarrs

Comment (saikumarrs) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx:88]: This is not a prominent or widely used feature that needs this kind of attention. So, in the favour of saving the page's real estate, let's merge it. However, let's keep the info message.

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**